VSTSAgent/DSCResources/xVSTSAgent/xVSTSAgent.psm1

```diff
@@ -171,6 +175,7 @@ function Set-TargetResource {
 
         if ( $Work ) { $installArgs['Work'] = $Work }
         if ( $LogonCredential ) { $installArgs['LogonCredential'] = $LogonCredential }
+        if ( $ProxyUrl ) { $installArgs['ProxyUrl'] = $ProxyUrl }
```

Comment:
Would be really nice to pass through the proxy username/password args as well.

Reply:
Unfortunately I have no means to test the combination: url + credential + password.